### DISCUSSION TITLE
建议大家多多宣传和推广这个项目

### DISCUSSION THREAD

**Discussion by freedom1933** [WeNeedHome/SummaryOfLoanSuspension#952] [Ideas]
为了让更多的人联合起来，希望大家在知乎、抖音、微博、网易等等社交平台，多宣传和推广这个项目，让更多的力量加入进来。
github，基本上是目前没有被墙，唯一可以集结的网站，请大家珍惜火种，利用好这个平台~~
另外，可以增加和996ICU项目互动。
同是天涯沦落人，受压迫的群体，应该联合起来，无论是程序员、制造业、服务业的人员，甚至是基层公务员。

**Comment by gaogaob** [comment]
👍
